hoomd/box.py

```diff
@@ -377,14 +382,10 @@ def volume(self):
     def volume(self, volume):
         self.scale((volume / self.volume)**(1 / self.dimensions))
 
-    @property
```

Comment:
This comment applies to line 355 (above).

Remove `lattice_vectors` to be consistent with the proposed change log entry.